server/api/story/__tests__/story.spec.js

```diff
@@ -24,45 +26,82 @@ test.serial('verify fixture database has stories', async t => {
   // Find all stories
   const p = await Story.find()
   t.is(t.context.stories.length, p.length)
-  // Find stories by things
+  // Find stories by name
   const q = await Story.findOne({ name: 'Voluntarily wins Open Source Web prize' })
   t.is(q && q.status, 'published')
 })
 
-test.beforeEach('connect and add story entries', async (t) => {
-  // save people
-  t.context.people = await Person.create(people).catch((err) => `Unable to create people: ${err}`)
-  // assign person as author
-  stories.map((story, index) => {
-    story.author = t.context.people[index]._id
-  })
-  t.context.stories = await Story.create(stories).catch((err) => console.error('Unable to create stories', err))
-})
-
-test.afterEach.always(async () => {
-  await Story.deleteMany()
-})
-
-test.serial('Should correctly give story 1 when searching by "wins"', async t => {
+test.serial('Should give list of all stories', async t => {
   const res = await request(server)
-    .get('/api/stories?search=wins')
+    .get('/api/stories')
     .set('Accept', 'application/json')
-    .set('Cookie', [`idToken=${jwtData.idToken}`])
     .expect(200)
     .expect('Content-Type', /json/)
   const got = res.body
-  t.is(stories[0].name, got[0].name)
-  t.is(2, got.length)
+  t.is(1, got.length)
 })
 
-test.serial('Should correctly give count of all stories sorted by name', async t => {
+test.serial('Should correctly give list of all stories belonging to parent', async t => {
   const res = await request(server)
-    .get('/api/stories')
+    .get('/api/stories/?parentId=5df0265916dc014f404ce0a0')
     .set('Accept', 'application/json')
-    .set('Cookie', [`idToken=${jwtData.idToken}`])
     .expect(200)
     .expect('Content-Type', /json/)
   const got = res.body
-  t.is(2, got.length)
-  t.is(got[0].name, 'Voluntarily wins Open Source Web prize')
+  t.is(1, got.length)
```

Comment:
good to check you get 1 result - but you also need to check it is the result you are expecting and not just the first one in the list. e.g t.is(got.parent,  parentId). probably best to add two stories with the same parent to the fixture.

Reply:
I got 1 result after merging 'beforeEach' with 'before'. 
I've added two stories with the same parent to the fixture and checked the got.parent result. The tests passed.